experimental/design-ios-gradle-dsl/build.gradle.kts

```diff
@@ -0,0 +1,65 @@
+plugins {
+    kotlin("multiplatform")
+    id("org.jetbrains.compose")
+}
+
+kotlin {
+    iosX64()
+    iosArm64()
+}
+
+compose.ios {
+    application {
+        mainClass = "Main_ios.kt"
+        bundleID = "com.example-company.example-app"
+        version = "1.2.3"
+
+        val distributionSigning = createSigningConfiguration {
+            name = "Distribution company signing"
```

Comment:
I suggest to keep signing configuration as similar to Mac desktop configuration as possible.
That means using the same property names & types, if possible.
I'm not sure if multiple configurations are necessary at this point: dev/production configuration could be achieved by simply having different Gradle properties on CI and locally.

Reply:
Thanks, good suggestion!
It will use existing configuration